OpenSpace/OpenSpace
# Create a tiny app for writing the documentation jsons

Add simple app for writing the documentation (only asset components and lua scripting).

This will be used by the Jenkins machine to automatically create the .json files for the documentation. It only works on windows and Linux.

To test:
* Use Cmake to generate a project where the docs writer is selected (by default it isn't)
* Set it as startup project in visual studio
* Run, and check the documentation folder in OpenSpace. It should have printed out two .json files, `scriptingApi.json` and `assetComponets.json`. 
* To test it super properly, place these json files in the OpenSpace-Docs/json folder, and then run `make html` (after activating the python virtual environment with `.\.venv\Scripts\activate`). This will generate the new documentation. Open it from the index file in the build folder. 
* See that it looks ok and that the console didn't print out error messages

Done!
